README.md

```diff
@@ -100,7 +113,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema has the definitions of the database
+
+one to many = teacher to students
 ```
```

Comment:
What exactly do you mean by "definitions"? What in particular does a schema file create in our database?